src/components/with-preview.js

```diff
@@ -36,20 +36,23 @@ export function WithPreview(klazz, updateData) {
     }
 
     componentDidMount() {
-      global.addEventListener("message", event => {
-        if (event.data && event.data.action == 'reloadStory' && event.data.story) {
```

Comment:
1. The bold issue occurs only for whoever is using the WithPreview component from quintype-node-components.
2. Initially we were getting `event.data.action` as `reloadStory`, but as we debugged we are now getting `sendStoryToIframe `.
3. Then it is recommended to test with this version.
4. We need to now test if the preview is working on tq/malibu for both home and story.
5. If it works, we will have to confirm with the platform team if `event.data.action` is set to `sendStoryToIframe`.(Akshay Gupta and Prabu has some context on this)
6. If not there are 2 solutions:
a. Platform will have to debug from where the `sendStoryToIframe` is coming from.
b. We will have to check and validate if only `event.data && event.data.story` checks are sufficient.(Confirm with Ram/Amogh)